Mathlib/Tactic/Inhabit.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2022 Joshua Clune. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joshua Clune
+-/
+import Lean
+
+open Lean.Meta
```

Comment:
```suggestion

/-!
Defines the `inhabit α` tactic, which tries to construct an `Inhabited α` instance,
constructively or otherwise.
-/

open Lean.Meta
```
Let's have module docs for every file!